pip-8210

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+
+if [[ ${-} != *i* ]]; then
+  source_once &> /dev/null && return 0
+fi
+
+#*# pip-8210
+
+#**
+# .. file:: pip-8210
+#
+# This script allows us to install a package outside of pip-tools. In order to accomplish correctly, we use the ``requirements.txt`` file as a constraint files,and this and handle corner cases like pip issue 8210.
+#
+# It add the -c constraints to ``pip install``, ``pip download``, and ``pip wheel``
```

Comment:
```suggestion
# Adds the -c constraints to ``pip install``, ``pip download``, and ``pip wheel``
```

Reply:
This doesn't really flow with anything else? What is it saying?

Reply:
```suggestion
# To accomplish this, it adds the ``-c`` constraints to ``pip install``, ``pip download``, and ``pip wheel`` sub-commands.
```